core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksInodeStore.java

```diff
@@ -86,18 +93,63 @@ public RocksInodeStore(String baseDir) {
     RocksDB.loadLibrary();
     mDisableWAL = new WriteOptions().setDisableWAL(true);
     mReadPrefixSameAsStart = new ReadOptions().setPrefixSameAsStart(true);
-    mIteratorOption = new ReadOptions().setReadaheadSize(
-        ServerConfiguration.getBytes(PropertyKey.MASTER_METASTORE_ITERATOR_READAHEAD_SIZE));
+    mIteratorOption = new ReadOptions().setTotalOrderSeek(true)
```

Comment:
Move assignments to field declaration.

Reply:
These must be called after RocksDB.loadLibrary()